app/js/directives/dateQuestion.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+angular.module('ddsApp').directive('dateQuestion', function(dateFormats, questionService) {
+    return {
+        restrict: 'E',
+        transclude: true,
+        templateUrl: 'partials/foyer/date-question.html',
+        scope: true,
+        controller: 'dateQuestionCtrl',
+        link: {
+            // We use pre, as the scope needs to be ready for sub-directive dds-date to compile
+            pre: function ($scope, $element, $attributes) {
+                questionService.bindModel($attributes.model, $scope);
+                $scope.max = $attributes.max;
+                $scope.min = $attributes.min;
+                $scope.format = $attributes.format;
```

Comment:
Ceci ne devrait-il pas être défini par [isolation du `scope`](https://code.angularjs.org/1.4.12/docs/guide/directive#isolating-the-scope-of-a-directive) ?